environment.yml

```diff
@@ -25,6 +25,7 @@ dependencies:
  - ruamel_yaml
  - gitpython
  - testpath>=0.4.2 # 0.4.1 is bad due to https://github.com/conda-forge/testpath-feedstock/issues/7
+ - tqdm
```

Comment:
their github page says that it is from conda-forge. are we ok with this?

Reply:
conda search seems to indicate that it's also in the main channel

Reply:
But this should be added to setup.py install_requires or the use made optional